content/600-about/02-style-guide/01-mdx-examples.mdx

```diff
@@ -121,6 +126,7 @@ model Post {
   author    User?   @relation(fields: [authorId], references: [id])
   authorId  Int?
 }
+
```

Comment:
Lines 141 to 162 should probably also be updated (manually) to match the new empty lines between blocks.